buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+#
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import base64
+import json
+import os
+import re
+import subprocess
+import sys
+import threading
+import yaml
+import urllib.request
+
+import bazelci
+from bazelci import BuildkiteException
+
+BUILD_STATUS_API_URL = "https://api.buildkite.com/v2/organizations/bazel/pipelines/bazelisk-plus-incompatible-flags/builds/"
+
+ENCRYPTED_BUILDKITE_API_TOKEN = """
+CiQA4DEB9ldzC+E39KomywtqXfaQ86hhulgeDsicds2BuvbCYzsSUAAqwcvXZPh9IMWlwWh94J2F
+exosKKaWB0tSRJiPKnv2NPDfEqGul0ZwVjtWeASpugwxxKeLhFhPMcgHMPfndH6j2GEIY6nkKRbP
+uwoRMCwe
+""".strip()
+
+INCOMPATIBLE_FLAGS = bazelci.fetch_incompatible_flags()
+
+class LogFetcher(threading.Thread):
+   def __init__(self, job):
+      threading.Thread.__init__(self)
+      self.job = job
+      self.log = None
+
+   def run(self):
+      self.log = get_build_log(self.job)
+
+
+def buildkite_token():
```

Comment:
You could refactor bazelci.saucelabs_token() to accept key and input, and then just call it here.

Reply:
I believe the SAUCELABS_TOKEN is different from BUILDKITE_API_TOKEN?